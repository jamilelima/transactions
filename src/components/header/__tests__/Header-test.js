import React from 'react';
import Header from '../index';
import {create} from 'react-test-renderer';

describe('Test Header component', () => {
  const header = create(<Header amountValue={200} />);
  test('Match with snapashot', () => {
    expect(header.toJSON()).toMatchSnapshot();
  });

  test('Test amountValue prop', () => {
    const instance = header.root;
    expect(instance.props.amountValue).toBe(200);
  });
});
