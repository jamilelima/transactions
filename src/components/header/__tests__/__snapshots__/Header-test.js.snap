// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Header component Match with snapashot 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#1B232F",
        "flexDirection": "row",
        "justifyContent": "space-between",
        "marginBottom": 0,
        "marginLeft": 0,
        "marginRight": 0,
        "marginTop": 0,
        "paddingBottom": 50,
        "paddingLeft": 50,
        "paddingRight": 50,
        "paddingTop": 50,
      },
    ]
  }
>
  <View
    style={
      Object {
        "marginRight": 50,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#FFF",
            "fontFamily": "Nunito-Regular",
            "fontSize": 16,
            "fontWeight": "bold",
            "textAlign": "center",
          },
        ]
      }
    >
      Segunda-feira
    </Text>
    <Text
      style={
        Array [
          Object {
            "color": "#FFF",
            "fontFamily": "Nunito-Regular",
            "fontSize": 16,
            "fontWeight": "bold",
            "textAlign": "center",
          },
        ]
      }
    >
      20/04/2020
    </Text>
  </View>
  <View>
    <Text
      style={
        Array [
          Object {
            "color": "#90949c",
            "fontFamily": "Nunito-Regular",
            "fontSize": 12,
            "textAlign": "justify",
          },
        ]
      }
    >
      Seu saldo
    </Text>
    <Text
      style={
        Array [
          Object {
            "color": "#FAFAFA",
            "fontFamily": "Nunito-Regular",
            "fontSize": 22,
            "fontWeight": "bold",
          },
        ]
      }
    >
      R$ 
      200.00
    </Text>
  </View>
</View>
`;
